src/button/index.js

```diff
@@ -5,7 +5,7 @@ import { compose, defaultProps } from 'recompose';
 import { body } from '../theme/fonts';
 import * as colors from '../theme/colors';
 
-const strokeColor = colors.C300
+const strokeColor = colors.N0
 
 const styles = {
   button: {
```

Comment:
I think @Dermah mentioned the current settings align better with the visuals on the style guide, rather than the numbers themselves.

Might be a `box-sizing` problem.

Also, what's the `transitionDuration` for?

Reply:
Yup, sorry about that. Thought these numbers were placed by mistake.
Added transition duration for hover effects (that users pass within the className prop).
Thought it would be good to have a consistent or default transition time.